.ci/run-link-check-plugin.sh

```diff
@@ -9,12 +9,26 @@ mvn --version
 curl --fail-with-body -I https://sourceforge.net/projects/checkstyle/
 mvn -e --no-transfer-progress clean site -Dcheckstyle.ant.skip=true -DskipTests -DskipITs \
    -Dpmd.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true
+mkdir -p .ci-temp
+
 echo "------------ grep of linkcheck.html--BEGIN"
 # "grep ... | cat" is required command is running in "set -e" mode and
 # grep could return exit code 1 if nothing is matching
-grep externalLink target/site/linkcheck.html | cat
+grep -E "doesn't exist|externalLink" target/site/linkcheck.html | grep -v 'Read timed out' \
+  |  cat > .ci-temp/linkcheck-errors.txt
+
+sort config/linkcheck-suppressions.txt > .ci-temp/linkcheck-suppressions-sorted.txt
+sort .ci-temp/linkcheck-errors.txt > .ci-temp/linkcheck-errors-sorted.txt
+
+# Suppressions exist until https://github.com/checkstyle/checkstyle/issues/11572
+diff .ci-temp/linkcheck-suppressions-sorted.txt .ci-temp/linkcheck-errors-sorted.txt || true
 echo "------------ grep of linkcheck.html--END"
-RESULT=$(grep externalLink target/site/linkcheck.html | grep -v 'Read timed out' | wc -l)
+
+RESULT=$(diff -y --suppress-common-lines .ci-temp/linkcheck-suppressions-sorted.txt \
+  .ci-temp/linkcheck-errors-sorted.txt | wc -l)
+rm .ci-temp/linkcheck-errors.txt
```

Comment:
Why? We have a fresh container every time we execute this, right?

Reply:
Uhh, I got curious if other scripts remove .ci-temp files, found some and decided to do it for some reason. Disregard. Removed.